extensions/ima/src/main/java/com/google/android/exoplayer2/ext/ima/AdTagLoader.java

```diff
@@ -1122,9 +1122,15 @@ private void handleAdPrepareError(int adGroupIndex, int adIndexInAdGroup, Except
   }
 
   private void ensureSentContentCompleteIfAtEndOfStream() {
+    if (player == null) {
```

Comment:
The `checkNotNull(player)` below suggests this condition isn't needed because the player should always be non-null here.